tests_integration/cli/shebang/sample-bin3

```diff
@@ -0,0 +1 @@
+#!/usr/bin/php
```

Comment:
It seems there's no test file associated with it? You'll  probably need to add a fake plugin to test it.

Something like:

```js
expect(
  prettier.getFileInfo.sync(
    "tests_integration/cli/shebang/sample-php",
    { plugins: [{ languages: [{
        name: "PHP",
        parsers: ["php"],
        interpreters: ["php"]
    }]}]}
  )
)
  .toMatchObject({
    ignored: false,
    inferredParser: "php"
  });
```

Reply:
Oops, I accidentally committed it after playing around a bit. It’d still be nice to have that, but unfortunately, adding your suggestion doesn’t work—something tries to look up the plugin from disk. For now, I’ll remove it.